packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -136,8 +146,18 @@ const findSelector = (component: (props: any) => JSX.Element | undefined): any =
       return participantsButtonSelector;
     case EndCallButton:
       return emptySelector;
+    case HoldButton:
```

Comment:
Possible to have a conditionally remove return statement instead of using an explicit if block above?

Reply:
we can't conditionally compile out the use of the HoldButton import if we do so we need the extra check